src/components/Article.jsx

```diff
@@ -20,7 +21,17 @@ function Article() {
       }
     };
 
+    const getComments = async () => {
+      try {
+        const commentData = await fetchCommentsById(articleId);
+        setComments(commentData);
+      } catch (error) {
+        console.error('Error fetching comments:', error);
```

Comment:
Please remove all logs. This part will be useful later for error handling to display a message on the page for the user.